### ISSUE TITLE
Do not render Non-PDF files in UI

### ISSUE THREAD

**Issue by Snify89** [eikek/docspell#1635]
You may upload an "incompatible" file to Docspell.
However when you open the item, the UI tries to render the file with no success, if it is not a PDF file.
Instead, the original file is automatically being downloaded, which is not desirable. Downloads should not start automatically if you open an item. The download also "sticks" to any other item and gets downloaded again! I might update this issue soon with a video presentation about this bug, if needed.

**Comment by eikek** [comment]
Yes I think this is a good idea. It could be made configurable, as some browsers could even display other files. For example, HTML can be displayed as well.
